README.md

```diff
@@ -2,22 +2,25 @@
 
 ## Requirements
 
-1. Postgres
+1. Yarn (https://classic.yarnpkg.com/en/docs/install/)
+1. Postgres (https://www.postgresql.org/download/)
    - when installing, use the default password "postgres"
    - for macOS users, suggested installation is through homebrew
 2. TypeORM Global Installation (`yarn add typeorm -g`)
    - this will install the TypeORM CLI as well
-3. Redis
+3. Redis (https://redis.io/download)
 
 ## Before starting
 
 1. Run `yarn install` to install all dependencies
 
 ## Configure environment variables
 
+Make a copy of `sample.env`, rename it to `.env`, and fill out the environment variables respectively.
+They're likely to be:
 ```
-Make a copy of `sample.env` and rename it to `.env`.
-Fill out the environment variables respectively
+DB_HOST=localhost:5432
+REDIS_HOST=localhost:6379
```

Comment:
Oh strange, you needed to specify the ports for the environment variables?
Mine just looks like this atm
![image](https://user-images.githubusercontent.com/49849754/107139933-9315ce00-68d3-11eb-946d-44f420b44a98.png)

Reply:
Mine works like that too ^!

Reply:
hmm maybe I didn't need to specify the port then haha